mews-api.md

```diff
@@ -563,9 +563,9 @@ There are certain rules that need to be followed in order for Mews to process th
   * The whole group is uniquely identified by `channelManagerId` in Mews and in the channel manager extranet.
   * Each reservation should have a unique code within the group. The same code for the reservation should be provided in any following modification message.
   * Each reservation in the group can have different `start`, `end`, `spaceTypeCode`, `ratePlanCode`.
-* Group total cost `group`.`totalCost` is the sum of each `reservation`.`totalCost`, which is the sum of all night costs and total costs of all `extras` of the `reservation`.
-  * All costs in the message are inclusive of VAT.
-  * If for any reason the `group`.`totalCost` is be different, Mews will automatically distribute the missing/additional cost to the nights, so the `group`.`totalCost` is achieved.
+* Group total cost `group`.`totalAmount` is the sum of each `reservation`.`totalAmount`, which is the sum of all night amounts and total amounts of all `extras` of the `reservation`.
+  * If for any reason the `group`.`totalAmount` is different, Mews will automatically distribute the missing/additional amount to the nights, so the `group`.`totalAmount` is achieved.
```

Comment:
Replace with,
"If for any reason the sum of the `reservation`.`totalAmount` is different, Mews will automatically distribute the missing/additional amount to the nights, so the `group`.`totalAmount` is achieved."